.github/workflows/bump-version.yml

```diff
@@ -1,6 +1,6 @@
 name: 'Bump Version'
 on:
-  push:
+  pull_request:
     branches:
```

Comment:
are we sure that this is the target branch and not the topic branch?

Reply:
Yes: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request